hw/ip/aes/model/crypto.h

```diff
@@ -5,6 +5,15 @@
 #ifndef CRYPTO_H_
 #define CRYPTO_H_
 
+/**
+ * AES cipher mode
+ */
+typedef enum crypto_mode {
```

Comment:
probably want to do the same 1<<0 thing here, you caught most (all?) of the other ones.

Reply:
Ups, I forgot about that. Thanks, it's fixed now.